### ISSUE TITLE
Winrm connections are not closed, resulting in multiple winrshost.exe and conhost.exe processes on target node

### ISSUE THREAD

**Issue by Robby123** [ansible/ansible#12910]
I am running a playbook which runs many tasks on remote windows nodes, however after each playbook completes I notice that there are still many 'winrshost.exe' and 'conhost.exe' processes running on the remote nodes. This causes winrm to crash after a while as it runs out of memory

**Comment by Robby123** [comment]
seems like it only happens when I use the 'raw' module

**Comment by jhawkesworth** [comment]
Which ansible version are you running please?

**Comment by Robby123** [comment]
2.0.0 0.0.pre

**Comment by jhawkesworth** [comment]
please can you try latest devel?  I recall someone reporting this a little while ago but I believe has been addressed fairly recently.

**Comment by trondhindenes** [comment]
@jhawkesworth correct, we saw this for a while, but should be resolved in devel.

**Comment by jhawkesworth** [comment]
@Robby123 - any update on this?

If it is still failing in latest devel then please can you share what you are attempting using 'raw' so this can be investigated further.

**Comment by gimoh** [comment]
I'm using 2.0.0-0.6.rc1 and I'm seeing this not only with raw module, but with win_chocolatey.  The playbook is just:

```
---
- hosts: default
  connection: winrm
  tasks:
    - name: install packages
      win_chocolatey:
        name: "{{ item }}"
      with_items:
        - 7zip.install
        - cmder
        - notepadplusplus.install
        - processhacker.install
```

After running it, there are multiple 'winrshost.exe' and 'conhost.exe' processes hanging about.

**Comment by cchurch** [comment]
Appears to be the same as #9980.

**Comment by cchurch** [comment]
@Robby123 This issue should be fixed by #13261.
